lighthouse-core/test/computed/manifest-values-test.js

```diff
@@ -141,6 +141,18 @@ describe('ManifestValues computed artifact', () => {
         const iconResults = results.allChecks.filter(i => i.id.includes('Icons'));
         assert.equal(iconResults.every(i => i.passing === false), true);
       });
+
+      it('fails when a manifest icon fails to fetch icon', async () => {
```

Comment:
I would've expected to see a test like this in installable-manifest.js since that audit failing is what we actually care about. move and/or copy something like it over there?

Reply:
done